vcd/resource_vcd_edgegateway.go

```diff
@@ -198,8 +218,20 @@ func resourceVcdEdgeGatewayCreate(d *schema.ResourceData, meta interface{}) erro
 		}
 
 		log.Printf("[TRACE] edge gateway load balancer configured")
+
+		log.Printf("[TRACE] edge gateway firewall configuration started")
+
+		err = updateFirewall(d, edge)
+		if err != nil {
+			return fmt.Errorf("unable to update firewall settings: %s", err)
+		}
+
+		log.Printf("[TRACE] edge gateway firewall configured")
+
 	}
 
+	// TODO double validate if we need to use partial state here
```

Comment:
Do we leave this?

Reply:
Yes. I intend to leave it for next iteration we touch it, because this is not a potential issue introduced by this PR and may be not an issue at all - this is more for consideration and ensuring that we either need it, either not.